coinsSimple.js

```diff
@@ -8706,6 +8706,92 @@ const coins = {
       "infura",
     ],
    },
+   coin: {
+    coin: "coin",
```

Comment:
Please change identifier name of coin to coinartisttoken (everywhere) . Having it as coin is confusing